restli-tools/src/test/extensions/invalidNoExtensionAnnotation/BazExtensions.pdl

```diff
@@ -0,0 +1,3 @@
+record BazExtensions includes Baz {
+  testField: array[DummyKey]
```

Comment:
I would leave a comment saying that this is invalid because extension annotations are required for 1-to-many relations.